src/main/java/org/kohsuke/github/GHGistFile.java

```diff
@@ -60,11 +64,16 @@ public String getLanguage() {
     }
 
     /**
-     * Content of this file.
+     * Content of this file. Lazy download the content in case the size is nonzero but the content is empty. Function
+     * modified for <a href="https://github.com/hub4j/github-api/issues/1325">issue #1325</a>.
      *
      * @return the content
      */
-    public String getContent() {
+    public String getContent() throws IOException {
```

Comment:
Sorry, adding an exception is considered an API change. You'll need to wrap the `IOException` in a `GHException` (which is a `RuntimeException` that can be thrown without being declared.  Add comments in the JavaDoc Explaining the situation.   
```suggestion
    public String getContent() {
```

You might add a new method  `public String getContentString() throws IOException {` and then have the existing `getContent()` wrap the new method.